test/e2e/network/service.go

```diff
@@ -1792,29 +1792,63 @@ var _ = SIGDescribe("Services", func() {
 		}
 	})
 
-	// [LinuxOnly]: Windows does not support session affinity.
-	ginkgo.It("should have session affinity work for service with type clusterIP [LinuxOnly]", func() {
+	/*
+		Release: v1.17
```

Comment:
these should probably be updated to v1.18, as v1.17 ended. Same for the rest below.